docs/en/reference/configuration.rst

```diff
@@ -302,10 +302,11 @@ pdo_oci / oci8
    database.
 -  ``host`` (string): Hostname of the database to connect to.
 -  ``port`` (integer): Port of the database to connect to.
--  ``dbname`` (string): Name of the database/schema to connect to.
+-  ``dbname`` (string): Name of the database/schema to connect to. Using this parameter is deprecated.
```

Comment:
So `dbname`, which is not marked as optional, is deprecated in favor of `servicename` or `sid`, both of which are optional. Is it mandatory to specify one of the two parameters?

Reply:
> Is it mandatory to specify one of the two parameters?

It's not. There may be the default service configured on the server. The previous documentation was too strict.